tests/swig/java/clean-test-files-java.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
```

Comment:
This file should not be here, cleanup should be done by autotest, and the deleted files should be added to a `.gitignore`.